scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -89,16 +92,28 @@ task SplitIntervals {
     }
   }
 
-  command {
+  command <<<
+    # Updating to use standard shell boilerplate
+    PS4='\D{+%F %T} \w $ '
+    set -o errexit -o nounset -o pipefail -o xtrace
     set -e
+
     export GATK_LOCAL_JAR=~{default="/root/gatk.jar" gatk_override}
 
+    WEIGHTED_BED_BEING_USED="~{using_weighted_beds}"
+    if [ -z "$WEIGHTED_BED_BEING_USED" ]
+    then
+      WEIGHT_BED_ARG=""
+    else
+      WEIGHT_BED_ARG="~{"--weight-bed-file " + interval_weights_bed}"
+    fi
+
     mkdir interval-files
     gatk --java-options "-Xmx~{java_memory}g" ~{gatkTool} \
       --dont-mix-contigs \
       -R ~{ref_fasta} \
       ~{"-L " + intervals} \
-      ~{"--weight-bed-file " + interval_weights_bed} \
+      $WEIGHT_BED_ARG \
```

Comment:
I don't understand this change, nor the need for the boolean `use_interval_weights` - the old syntax should work when passed an optional parameter - that is it will expand the line to `--weight-bed-file foo` if foo is defined, or else it will just have an empty line there. See https://github.com/broadinstitute/warp/blob/1a1032ddc447baa69ed25b0604a818f541f1c9e4/tasks/broad/BamProcessing.wdl#L102

Reply:
The code appears to have been intended to run the GATK tool SplitIntervals or WeightedSplitIntervals depending on whether the interval weights bed was defined.  In practice, because we were always using WeightedSplitIntervals, the path for SplitIntervals was broken.  Passing in  --weight-bed-file caused a hard failure as it didn't recognize that argument.  In order to make that logic function as intended, I needed that entire line to be empty in the case where we were intending to use SplitIntervals

Reply:
with the change that allowed me to pass down a proper undefined interval_weights_bed, this DID resolve to an empty string.  I no longer need this hacky code!